packages/e4s_cl/cli/commands/profile/list.py

```diff
@@ -5,7 +5,7 @@
 
 .. code::
 
-    $ e4s-cl profile list s
```

Comment:
This is an example of name filtering in `profile list`, although the actual example is poorly chose, it should still be accounted for

Reply:
New example has been provided to avoid confusion